README.rst

```diff
@@ -282,7 +282,7 @@ A general live API documentation (following the OpenAPI specification) at ``/api
 Browsable web interface
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: https://github.com/openwisp/openwisp-users/raw/master/docs/images/api-ui.png
+.. image:: https://github.com/openwisp/openwisp-users/raw/docs/docs/images/api-ui.png
```

Comment:
I have one minor request here, can we show something more meaningful here? Eg: the output of /api/v1/users/user/, showing some users.

Before the only endpoint was this one, now we have more endpoints and we can show something better than this.

Reply:
Please See: https://github.com/openwisp/openwisp-users/blob/docs/docs/images/api-ui.png

Reply:
Please see: https://raw.githubusercontent.com/openwisp/openwisp-users/docs/docs/images/api-ui.png